Cargo.toml

```diff
@@ -17,6 +17,6 @@ nightly = []
 [dependencies]
 crossbeam = "0.3.0"
 crossbeam-epoch = "0.2.0"
-crossbeam-utils = "0.1.0"
+crossbeam-utils = "0.2.0"
```

Comment:
For reasons I discussed in https://github.com/crossbeam-rs/crossbeam-epoch/issues/58, I'd like to write `crossbeam-utils = "0.2.1"`.  cc @stjepang @Amanieu